docker-compose.yml

```diff
@@ -57,13 +57,15 @@ services:
     depends_on:
       - bancho
     volumes:
-      - ./ext/nginx.conf:/etc/nginx/conf.d/default.conf:ro
+      - ./ext/docker-nginx.conf.example:/etc/nginx/templates/nginx.conf.template:ro
```

Comment:
very odd choice to use an "example" file in here, when the .env file is with the normal name

Reply:
the filename was changed in #425 so I used the file as is

Reply:
That PR is a bit problematic

# create a configuration file from the sample provided
cp .env.example .env

this is now incorrect in both setup methods (+ translations)

also, the example cannot remain in the compose. i propose adding instructions to do something similar to .env, or to just rename the file and move it in `docker/` (could be the better solution)